dask/dataframe/utils.py

```diff
@@ -657,6 +657,12 @@ def index_summary(idx, name=None):
     return "{}: {} entries{}".format(name, n, summary)
 
 
+def series_like_df(df, values=()):
```

Comment:
Can we make a couple change here? Can this just return the type / constructor for df? And can we update the name to reflect that? I'm finding the usage of this really confusing because it does the series-type stuff and the series construction stuff (but you can't pass name / index, just values).

```python
def series_type_like_df(df):
    """Get the concrete series type compatible with df."""
    # TODO: https://github.com/pandas-dev/pandas/issues/27824
    # Use a standard API, which will handle empty DataFrames.
    return type(df._meta.iloc[:, 0])
```

Then update uses of this to be like `divisions = series_like_df(df)(divisions)`

Reply:
Agreed - Sorry about the ugliness here.  I was actually planning to submit a PR to cudf (to add `_constructor_sliced`) and ultimately remove this.. Sorry for the delay with this.

Reply:
No worries. We'll maybe need something like this for a while, depending on how that pandas issue is resolved.